phpunit.xml.dist

```diff
@@ -29,7 +29,7 @@
 	</filter>
 
 	<logging>
-		<log type="coverage-clover" target="tests/reports/clover.xml" />
+		<log type="coverage-clover" target="tests/coverage/php/clover.xml" />
```

Comment:
Moved coverage reporting into dedicated directories for PHP and JS, which is required for Coveralls reporting since both generate their own set of coverage files.